_usage/locales.md

```diff
@@ -19,109 +19,117 @@ encoding. Babelfish will automatically translate from the server
-| `sql_latin1_general_cp1251_ci_as` | [Deterministic single-byte character encoding](https://www.unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WindowsBestFit/bestfit1251.txt) that provides support for Latin characters. |
-| `sql_latin1_general_cp1_ci_as` | [Deterministic single-byte character encoding](https://www.unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WindowsBestFit/bestfit1252.txt) that provides support for Latin characters.  |
-| `sql_latin1_general_cp1253_ci_as` | [Deterministic single-byte character encoding](https://www.unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WindowsBestFit/bestfit1253.txt) that provides support for Latin characters.   |
-| `sql_latin1_general_cp1254_ci_as` | [Deterministic single-byte character encoding](https://www.unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WindowsBestFit/bestfit1254.txt) that provides support for Latin characters. |
-| `sql_latin1_general_cp1255_ci_as` | [Deterministic single-byte character encoding](https://www.unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WindowsBestFit/bestfit1255.txt) that provides support for Latin characters.   |
-| `sql_latin1_general_cp1256_ci_as` | [Deterministic single-byte character encoding](https://www.unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WindowsBestFit/bestfit1256.txt) used to write languages that use Latin characters. |
-| `sql_latin1_general_cp1257_ci_as` | [Deterministic single-byte character encoding](https://www.unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WindowsBestFit/bestfit1257.txt) that provides support for Latin characters. |
-| `sql_latin1_general_cp1258_ci_as` | [Deterministic single-byte character encoding](https://www.unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WindowsBestFit/bestfit1258.txt) used to write Latin characters. |
-| `chinese_prc_ci_as`  | Provides support for Chinese (PRC).  |
-| `cyrillic_general_ci_as` | Provides support for Cyrillic. |
-| `finnish_swedish_ci_as` | Provides support for Finnish.  |
-| `french_ci_as` | Provides support for French. |
-| `korean_wansung_ci_as` | Provides support for Korean (with dictionary sort) |
-| `latin1_general_ci_as` | Provides support for Latin characters. |
-| `modern_spanish_ci_as` | Provides support for Modern Spanish. |
-| `polish_ci_as` | Provides support for Polish. |
-| `thai_ci_as` | Provides support for Thai. |
-| `traditional_spanish_ci_as` | Provides support for Spanish (Traditional sort) |
-| `turkish_ci_as`  | Provides support for Turkish. |
-| `ukrainian_ci_as`   | Provides support for Ukranian.   |
-| `vietnamese_ci_as`  | Provides support for Vietnamese. |
-
-You can use the following collations for database objects:
-
-| Dialect   | Deterministic options | Non-deterministic options         |
+| `bbf_unicode_general_cs_as` | Case sensitive collation for the UTF-8 encoding that is designed to work as well as possible in multi-language environments. |
```

Comment:
these CS_AS collations may not be specified as the server_collation_name, but any Babelfish collation may be specified on a column or expression

Reply:
Are you sure? I just created a Babelfish database with that collation, and there was no error.

If that is indeed not allowed, could you please provide a complete list of available collation names?

Reply:
it's all the babelfish collations that have ci_as in their names, which I believe is exactly the same as the code above that was deleted.